keyboards/bear_face/v1/keymaps/default/keymap.c

```diff
@@ -28,13 +14,13 @@ enum layers {
 
 //custom keycode enums
 enum custom_keycodes {
-    BASE_QWER = SAFE_RANGE,
+    BASE_QWER = QK_KB_0,
```

Comment:
This change is logically incorrect. Keymaps should use the keymap block of keycodes, which is `QK_USER`/`SAFE_RANGE`  where the first keycode is `QK_USER_1`.